examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb

```diff
@@ -0,0 +1,1316 @@
+{
```

Comment:
<p>Should train be true for loading this data given it is for testing?</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>